front/packages/ui/src/home/recommanded.tsx

```diff
@@ -152,19 +154,20 @@ export const ItemDetails = ({
 					{...css({
 						bg: (theme) => theme.themeOverlay,
 						flexDirection: "row",
+						pX: 4,
 						justifyContent: "space-between",
 						minHeight: px(50),
 					})}
 				>
-					{(isLoading || genres) && (
```

Comment:
doesn't that result in skeletons being always shown for collections?

Reply:
Nope :), this is what the change two lines below is for

Reply:
Ok, I just saw that, but you disabled skeletons for everything. `Array(3).map(() => 2)` result in an empty array. Array(3) create an empty array with a capacity of 3 so .map() returns an empty array (that's why you need [...Array(3)] instead.

Reply:
Solved by using 'flex-end' on the parent component